src/run.ts

```diff
@@ -1,25 +1,27 @@
+/* eslint-disable multiline-comment-style */
```

Comment:
I hate this one too
It's so much easier to deal with comments as `//`

Reply:
It would be nice if we could flip it in all the `.eslintrc.yaml` files.